lib/terrapin/command_line/multi_pipe.rb

```diff
@@ -39,11 +39,11 @@ def close_read
       end
 
       def read_stream(io)
-        result = ""
+        result = []
```

Comment:
Thanks. What do you think about using `String.new` to create a mutable string here instead of an array and a join?

```suggestion
        result = String.new
```